elytron/WFCORE-5691-Bearer-Token-Authorization-Timeout.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+  - elytron
+---
+= [Preview] Bearer token timeout configurability will be added to WildFly's Elytron subsystem.
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:
+
+== Overview
+
+Bearer Token Authorization is the process of authorizing HTTP requests based on
+the existence and validity of a bearer token.  The token carries within it
+an expiration timestamp.  The two parameters being added, connection-timeout
+and read-timeout are placed on the URL when obtaining the public key from the OAuth2 provider.
```

Comment:
What public key?

What does placing these parameters on the URL do?

A user story would be good here.